LSP.sublime-settings

```diff
@@ -172,6 +172,9 @@
   // customize the color.
   "document_highlight_style": "underline",
 
+  // Enable highlighting on multiline blocks
+  "document_highlight_multiline": true,
```

Comment:
There is `show_multiline_diagnostics_highlights` already so it would be good to keep the naming consistent with it - `show_multiline_document_highlights`.